Automattic/woocommerce-payments
# Spike: optimize order update on merchant store after platform checkout payment

Fixes #

#### Changes proposed in this Pull Request









This is an exploration of optimizing the request we currently use to (1) update the order on the merchant store; and (2) get the merchant store return URL.

- If a request from the platform checkout is detected we only do the work required there.

#### Testing instructions





* Make sure platform checkout works the same before and after this PR has been applied.
    * This branch is only required on the merchant store but should work regardless of whether this PR is applied on the platform checkout site or not.

-------------------


**Post merge**
